components/event_dispatcher.rst

```diff
@@ -135,7 +135,7 @@ A call to the dispatcher's ``addListener()`` method associates any valid
 PHP callable to an event::
 
     $listener = new AcmeListener();
-    $dispatcher->addListener('acme.action', array($listener, 'onFooAction'));
+    $dispatcher->addListener('foo.action', array($listener, 'onFooAction'));
```

Comment:
FOS change password action have some sub-event possibilities, it makes more sense to nest these as you point here than the example show into the event dispatcher one.

To use `'acme.foo.action'` I consider using a pattern like `'{vendor}.{context}.{action}'`. Do you agree with this before I update?